_posts/2016-08-08-facts-about-publishing-open-source-code-in-government.md

```diff
@@ -70,12 +70,12 @@ There are several agencies that do classified work and release code that isn’t
+Just today, [the White House released](https://www.whitehouse.gov/blog/2016/08/08/peoples-code) its [federal source code policy](https://sourcecode.cio.gov/) ([PDF link](https://www.whitehouse.gov/sites/default/files/omb/memoranda/2016/m_16_21.pdf)) that requires agencies to begin developing policies that will require releasing a portion of custom-developed code as public open source. In other words: most agencies will be required to do some open source work, so it’s helpful to start work on this soon. Prior to this, the federal government built many important projects using open source code. The Consumer Financial Protection Bureau built an open source [tool](http://www.consumerfinance.gov/find-a-housing-counselor/) using open data from the Department of Housing and Urban Development to find nearby free housing counselors for those struggling to make mortgage or rent payments. 18F and the Presidential Innovation Fellows built [notalone.gov](https://notalone.gov/) to help survivors of sexual assault can find resources and data. The [College Scorecard](https://collegescorecard.ed.gov/) and its underlying data API were also all built with open source code.
 
 
 Here are a few federal explanations of the benefits of open source development:
 
-- The [draft federal source code](https://sourcecode.cio.gov/) policy discusses how code reuse can have significant benefits for American taxpayers, such as reducing vendor lock-in, decreasing duplicative costs for the same code, and increasing transparency across the federal government.
+- The [federal source code policy](https://sourcecode.cio.gov/) discusses how code reuse can have significant benefits for American taxpayers, such as reducing vendor lock-in, decreasing duplicative costs for the same code, and increasing transparency across the federal government.
```

Comment:
Just did: yes.

(I changed it because sourcecode.cio.gov isn't the draft policy anymore, so _something_ should change about the sentence.)